pkg/apis/agones/v1/fleet.go

```diff
@@ -237,3 +237,26 @@ func SumStatusReplicas(list []*GameServerSet) int32 {
 
 	return total
 }
+
+// SumSpecReplicas returns the total number of
+// Spec.Replicas in the list of GameServerSets
+func SumSpecReplicas(list []*GameServerSet) int32 {
```

Comment:
Should these functions have their own Unit tests?

Reply:
Yes, that's true, adding those tests.